src/main/java/com/jcabi/dynamodb/maven/plugin/RunMojo.java

```diff
@@ -29,77 +29,41 @@
  */
 package com.jcabi.dynamodb.maven.plugin;
 
-import java.io.File;
 import java.io.IOException;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
 import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
-import org.apache.maven.plugins.annotations.Parameter;
 
 /**
- * Starts DynamoDB Local.
- *
- * @todo #41:30min Let's avoid code duplication between this class and `StartMojo`.
- *  One idea is to create a decorator called `ThreadedMojo` that receives
- *  another mojo in its constructor and, when called, runs it inside a thread.
+ * Runs DynamoDB Local.
  *
  * @author Denis N. Antonioli (denisa@sunrun.com)
+ * @author Simon Njenga (simtuje@gmail.com)
  * @version $Id$
  * @since 0.8
+ * @todo #41:30min Let's avoid code duplication between this class and `StartMojo`.
```

Comment:
@simonjenga the puzzle must  be removed
btw why you didn't create a decorator as the puzzle creator suggested?